src/language_server_protocol.rs

```diff
@@ -1504,13 +1504,25 @@ impl LanguageClient {
         if help.signatures.is_empty() {
             return Ok(Value::Null);
         }
+
+        // active_signature may be negative value.
+        // So if it is negative value, we convert it into zero.
+        let active_signature_index =
+            (help.active_signature.unwrap_or(0).max(0) as u64).to_usize()?;
```

Comment:
You can cast to `usize` immediately after the `max` call, which saves an conversion.